index.html

```diff
@@ -0,0 +1,144 @@
+        <li class="menu-item"><a href="#">Locations</a></li>
+        <li class="menu-item"><a href="#">How to Book</a></li>
+        <li class="menu-item"><a href="#">Pricing</a></li>
+        <li class="menu-item"><a href="#">Cleaning Tips</a></li>
+      </ul>
+    </nav>
+  </header>
+  <section class="container-fluid main-content">
+    <div id="section-1">
+      <h4>"Cleanliness is next to Godliness"</h4>
+      <h1 class="title">HIRE TRUSTED HOUSEKEEPERS READY TO CLEAN</h1>
+      <p>
+        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Consequatur,
+        dolores ipsa quo sunt omnis nemo autem corrupti minima sapiente
+        veritatis rem culpa consectetur cupiditate vitae amet aliquam non
+        necessitatibus iste.
+      </p>
+      <h2 class="c-hours">CLEANING HOURS - 9:00 AM TO 5:00 PM (MON - FRI)</h2>
+      <p class="c-msg">We have the best offers available on the market</p>
+    </div>
+  </section>
+  <section id="section-2">
+    <h4>Our Services</h4>
+    <hr class="lowerBar">
+    <div class="services">
+      <div class="service-list">
+        <img src="./assets/images/washing-machine-nogb.png" alt="" />
+        <h4>Laundry Services</h4>
+        <p>
+          We undertake laundry services for both Domestic and commercial
+          facilities
+        </p>
+      </div>
+      <div class="service-list">
+        <img src="./assets/images/householdcores-nobg.png" alt="" />
+        <h4>Household Chores</h4>
+        <p>
+          Tired of domestic household chores or simply need a helping! We've got you covered
+        </p>
+      </div>
+      <div class="service-list">
+        <img src="./assets/images/swimmingpool-nobg.png" alt="" />
+        <h4>Swimming Pool Cleaning</h4>
+        <p>
+          Keep your swimming clean and hygienic. We provide industry leading pool solutions
+        </p>
+      </div>
+      <div class="service-list">
+        <img src="./assets/images/landscaping-nobg.png" alt="" />
+        <h4>Landscaping</h4>
+        <p>
+          Indulge yourself! Lets us provide you with a captivating and beautiful scenery every time you step out into your compund
+        </p>
+      </div>
+      <div class="service-list">
+        <img src="./assets/images/handyman.svg" alt="" />
+        <h4>Handyman Services</h4>
+        <p>
+          Worried about keeping basic equipment and other facilities in good working conditions! We are just a call away
+        </p>
+      </div>
+    </div>
+    <div class="contact-us">
+      <a href="http://"><span class="contact">Contact Us Today</span></a>
+    </div>
+    <h3><a href="#">SEE OUR COMPLETE LIST OF SERVICES</a></h3>
+  </section>
+  <section id="section-3">
+    <h3>Our Staff</h3>
+    <hr class="lowerBar">
+
+    <div id="stafflist">
+
+    </div>
+    <button type="button" class="more btn btn-outline-dark btn-lg">More <span class="">&#8744;</span>
+    </button>
+  </section>
+  <section id="section-4">
+    <h3>Partners</h4>
```

Comment:
Kindly fix the typo here, the opening and closing tags should be the same, either `h3` or `h4` 🤗